client/forest-manager/src/in_memory.rs

```diff
@@ -0,0 +1,30 @@
+pub struct InMemoryForestStorage {}
+
+impl InMemoryForestStorage {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+impl storage_hub_infra::storage::ForestStorage for InMemoryForestStorage {
```

Comment:
Wouldn't it be more natural if the `ForestStorage` trait was defined in the `forest-manager` crate? Could be in `lib.rs` or in some specific file for it.